app/cni/pkg/cni/main.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/containernetworking/cni/pkg/skel"
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/containernetworking/cni/pkg/types/current"
+	types100 "github.com/containernetworking/cni/pkg/types/100"
```

Comment:
Why not keeping current this way ?
```suggestion
	current "github.com/containernetworking/cni/pkg/types/100"
```

As it is done in [the cni documentation](https://github.com/containernetworking/cni/blob/v1.0.1/Documentation/spec-upgrades.md#specific-guidance-for-plugins-written-in-go)

Reply:
makes sense